Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM bigcommerce/ruby:2.0.0-p576
+FROM hub.services.bigcommerceapp.com/bigcommerce/ruby:2.0.0-p576
```

Comment:
Should probably also use a more recent version of Ruby.
On 27 Aug 2015 15:50, "Grégory Horion" <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/bigcommerce/hello-world-app-ruby-sinatra/pull/12#discussion_r38155728>
> :
>
> > @@ -1,4 +1,4 @@
> > -FROM bigcommerce/ruby:2.0.0-p576
> > +FROM hub.services.bigcommerceapp.com/bigcommerce/ruby:2.0.0-p576
>
> we should probably start from a public image... what do you
> @bigcommerce/ruby-sf <https://github.com/orgs/bigcommerce/teams/ruby-sf>
> think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bigcommerce/hello-world-app-ruby-sinatra/pull/12/files#r38155728>
> .
>